api/apps/api/src/modules/scenarios/scenario.api.entity.ts

```diff
@@ -94,6 +94,10 @@ export class Scenario extends TimeUserEntityMetadata {
   @Column('uuid', { name: 'project_id' })
   projectId!: string;
 
+  @Column('integer', { name: 'project_scenario_id' })
+  @IsOptional()
+  projectScenarioId?: number;
```

Comment:
this should be not null, at least once we've run a migration to give a number to each of the scenarios already in an instance at the time we add this new column (so I think it should be created as null, and then altered to not null, but the entity should have this as always defined)